accepted/2020/workloads/workloads.md

```diff
@@ -0,0 +1,92 @@
+* Users can install workloads via any supported installation tool, such as the VS Installer, a Linux package manager, or the CLI.
+
+## High-level approach
+
+The proposal can be broken into two main topics: acquisition, and composition.
+
+* *Acquisition* describes the capability and mechanism to acquire a workload, in part or in full, via an installation tool.
+* *Composition* describes file formats, pack types, on-disk locations, and other mechanisms that define a workload, constituent parts, and their associated versions, that enable them to be acquired, updated, and used as a coherent set.
+
+Both of these topics have been satisfied by the monolithic SDK, to date, and need to be reconsidered given the plan to add new, optional, workloads. A new approach to composition and acquisition will be discussed in the following sections.
+
+The key learning of .NET Core 1.x was that the .NET platform needs a separate versioning model than the one offered for NuGet library packages. A case in point is that you can update a single target framework value in a project, from `netcoreapp2.1` to `netcoreapp3.1`, to update all dependent platform components automatically (like for ASP.NET Core). The use of a single version for the platform makes it easy to update a project, and results in a stable configuration that is (for the most part) unaffected by package references.
+
+This means that optional components will not become NuGet packages that require package references in project files, or that are acquired as part of package restore. Instead, a new system will be developed that is an evolution of the SDK composition system, that evolves from a physically monolithic SDK to a logically monolithic one. This approach will enable us to deliver SDK UX with .NET 5 that is similar in spirit to the .NET Core 3.1 SDK, with the acknowledgment that there is some regression in functionality (you don't get the kitchen and its sink by default).
+
+Since the first .NET Core release, we have delivered various types of "packs" as specialized packages on NuGet feeds ([example from 1.x and 2.x era](https://www.nuget.org/packages/runtime.win-x64.Microsoft.NETCore.App/); [example from 3.x+ era (unlisted)](https://www.nuget.org/packages/Microsoft.NETCore.App.Runtime.linux-x64)). We use NuGet for this purpose because many tools and services understand NuGet packages, it includes a concept of feeds (which can be private or public), and offers a global CDN (in the case of NuGet.org). These packs, however, are not intended to be directly referenced by project files, but are known to the .NET Core SDK as an implementation detail. We intend workloads to follow a similar model, with the difference that the .NET SDK will not have detailed built-in knowledge of packs, and that workloads will be separately acquirable and updatable from the SDK.
```

Comment:
I'm not sure this sufficiently distinguishes packs and normal packages. Aren't packs an implementation detail (a very convenient one)

Reply:
This paragraph would be more helpful if it came after introducing the relationship between workloads and packs. It's less important how we are using NuGet in a different way than how workloads and packs relate.

Reply:
Packs are an implementation detail of the SDK but a key part of this proposal. The doc states that they are specialized packages and (at least going forward) unlisted. That seems to be enough detail for the overview.

At this point of the doc, I haven't introduced how workload and packs relate since I'm talking about our practice to date, where we don't really have this distinction.